x/stake/types/delegation.go

```diff
@@ -9,6 +9,19 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
```

Comment:
Do you think we can come up with more useful names for these two structs? One is for delegations and other is for redelegations, right?

Reply:
For now, but I think in general they can be used as "marshalable pointers" for getting other stuff from state too, other than just unbonds and redelegations.